io.catenax.id_based_request_for_update/1.0.0/IdBasedRequestForUpdate.ttl

```diff
@@ -0,0 +1,94 @@
+:weekBasedCapacityGroupList a bamm:Property;
+    bamm:description "Capacity groups the customer requests an update for."@en;
+    bamm:characteristic :WeekBasedCapacityGroupCharacteristic;
+    bamm:preferredName "Week Based Capacity Group List"@en.
+:WeekBasedCapacityGroupCharacteristic a bamm-c:List;
+    bamm:description "A list of capacity group UUIDs an update is requested for."@en;
+    bamm:preferredName "Week Based Capacity Group Characteristic"@en;
+    bamm:dataType :WeekBasedCapacityGroupEntity.
+:WeekBasedMaterialDemandEntity a bamm:Entity;
+    bamm:description "Encapsulates the information that is necessary for an update request."@en;
+    bamm:preferredName "Week Based Material Demand Entity"@en;
+    bamm:properties (:materialDemandId :changedAt).
+:capacityGroupID a bamm:Property;
+    bamm:description "The Capacity Group ID uniquely identifies the capacity group the supplier requests an update for within the business relationship between the supplier and its customer."@en;
+    bamm:characteristic :UUIDv4IdTrait;
+    bamm:exampleValue "0157ba42-d2a8-4e28-8565-7b07830c1110";
+    bamm:preferredName "Capacity Group ID"@en.
+:UUIDv4IdTrait a bamm-c:Trait;
+    bamm:description "Trait to ensure data format for UUIDv4."@en;
+    bamm:preferredName "UUIDv4 Trait"@en;
+    bamm-c:baseCharacteristic :UUIDv4;
+    bamm-c:constraint :UUIDv4RegularExpression.
+:changedAt a bamm:Property;
+    bamm:description "Point in time when the content (any property according to the data model) of the entity UUID was changed (according to information provided by the data provider), either by a human user or an automated process."@en;
+    bamm:characteristic :Timestamp;
+    bamm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp;
+    bamm:preferredName "Changed At"@en.
+:Timestamp a bamm:Characteristic;
+    bamm:description "Describes a Property which contains the date and time with an optional timezone."@en;
+    bamm:preferredName "Timestamp"@en;
+    bamm:dataType xsd:dateTime.
+:UUIDv4 a bamm:Characteristic;
+    bamm:description "A version 4 UUID is a universally unique identifier that is generated using random 32 hexadecimal characters."@en;
+    bamm:preferredName "UUIDv4"@en;
+    bamm:dataType xsd:string.
+:UUIDv4RegularExpression a bamm-c:RegularExpressionConstraint;
+    bamm:description "The provided regular expression ensures that the UUID is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens)."@en;
+    bamm:preferredName "UUIDv4 Regular Expression"@en;
+    bamm:value "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$".
+:WeekBasedCapacityGroupEntity a bamm:Entity;
+    bamm:description "Encapsulates the information that is necessary for an update request."@en;
+    bamm:preferredName "Week Based Capacity Group Entity"@en;
+    bamm:properties (:capacityGroupID :changedAt).
```

Comment:
In the new model request, changedAt was shown as optional property. Has this changed?